Assets/MixedRealityToolkit/Inspectors/MixedRealityShaderGUI.cs

```diff
@@ -0,0 +1,274 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.Rendering;
+using Object = UnityEngine.Object;
+
+namespace Microsoft.MixedReality.Toolkit.Editor
+{
+    /// <summary>
+    /// A custom shader inspector for the "Mixed Reality Toolkit/Wireframe" shader.
```

Comment:
>A custom shader inspector for the "Mixed Reality Toolkit/Wireframe" shader. [](start = 8, length = 75)

I think this was some copy pasta from the wireframe shader -maybe something like "A base shader inspector for other MRTK shaders"?